src/app/public/modules/action-hub/action-hub.component.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { TestBed } from '@angular/core/testing';
+import { expect } from '@skyux-sdk/testing';
+
+import { SkyActionHubComponent } from './action-hub.component';
+import { SkyActionHubModule } from './action-hub.module';
+
+describe('Action hub component', async () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [SkyActionHubModule]
+    }).compileComponents();
+  });
+
+  it('should show the title', () => {
+    const fixture = TestBed.createComponent(SkyActionHubComponent);
```

Comment:
Minor - any reason we couldn't pull the `fixture` into a variable declared in the `describe` and then move the `TestBed.createComponent` to the `beforeEach`? That is what we do in most of our components to avoid having to have the same line in each test.

Reply:
It appears you are doing this in the `link-list` spec.